src/js/profile/wearable/widget/wearable/DatePicker.js

```diff
@@ -130,6 +130,14 @@
 				self._circleIndicatorSupporter = null;
 				self._activeSelector = null;
 				self._spins = {};
+
+				// property contains information which DOM elements
+				// was built during widget build process.
+				// These data are needed in destroy method
+				self._wasBuilt = {
```

Comment:
As I see all properties of this object would be always set to false.
However widget "set" button is placed inside <footer> which is a child of `div.ui-date-picker' (`this.element`), and its inner HTML is cleared in destroy method.

Currently this code has no effect (but I think it would be safer to put footer inside page instead of `div.ui-date-picker`)

Please consider removing this code, or moving footer and its button into page.

Reply:
Done.